src/zerocash/utils/util.cpp

```diff
@@ -82,9 +84,7 @@ void printBytesVectorAsHex(const std::string str, const std::vector<unsigned cha
 }
 
 void getRandBytes(unsigned char* bytes, int num) {
-    int ret = RAND_bytes(bytes, num);
-    if(ret != 1)
-        std::cout << "rand_bytes error!" << ERR_get_error() << std::endl;
+    randombytes_buf(bytes, num);
```

Comment:
Actually I looked at the source code and it can fail, when it does it calls `abort()`.